dynamix/correlator/common.py

```diff
@@ -0,0 +1,208 @@
+import numpy as np
+from os import linesep
+
+import pyopencl.array as parray
+from pyopencl.tools import dtype_to_ctype
+from silx.opencl.common import pyopencl as cl
+from silx.opencl.processing import OpenclProcessing, KernelContainer
+
+from ..utils import get_opencl_srcfile
```

Comment:
You can use the silx resource machinery when providing a prefix like in "dynamix:openCL/src_filename.cl"